### ISSUE TITLE
HTML entities are processed twice

### ISSUE THREAD

**Issue by bburhans** [johnknoop/vscode-handlebars-preview#13]
The preview pane is processing HTML output an extra time before rendering it in version 1.0.1 using latest stable VS Code on Windows 10 (18362). If there is a preview pane setting I'm missing, or this is caused by a known flaw in VS Code, please let me know and accept my apologies.

Demonstration of the unexpected behavior:

![image](https://user-images.githubusercontent.com/311277/70554346-f2027a80-1b4a-11ea-8cc5-ac090be3bb33.png)

Compare to http://tryhandlebarsjs.com/, which behaves as expected:

![image](https://user-images.githubusercontent.com/311277/70554412-1199a300-1b4b-11ea-8ac2-339545a3dfcb.png)

![image](https://user-images.githubusercontent.com/311277/70554436-1cecce80-1b4b-11ea-8e92-3df7220d5ad3.png)

Input code for both of the above so you can copy and paste to reproduce:
```hbs
<p>
  foo &lt;bar&gt; baz
</p>

<p>
  foo &amp;lt;bar&amp;gt; baz
</p>
```

**Comment by johnknoop** [comment]
I'm not sure I understand what you mean, Your template contains two paragraphs, and both are rendered. What were you expecting to see?

**Comment by johnknoop** [comment]
Issue solved in commit 736b1cf69efcf7c70c79acc04b67b4546bd3cf98.

- [ ] Publish to marketplace

**Comment by johnknoop** [comment]
There's now a v 1.0.2 on VS Code marketplace.
